### PR TITLE
README: document "more interesting" aspects of the GitForWindowsHelper App

### PR DESCRIPTION
**Pull request by dscho**
So far, the README's audience was developers interested to help developing the GitHub App.

With this PR, the README will actually start out with the motivation for this App, followed by documentation for the slash commands that are implemented.

In other words, this PR expands the README's audience to include users of the App, as well as people who are merely interested in learning how the Git for Windows project is run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,56 @@
...
+
+The GitForWindowsHelper GitHub App supports so-called "slash commands", i.e. commands that start with a forward slash and that are issued via comments in GitHub Issues or Pull Requests.
+
+### `/hi`
+
+**Where can it be called?** In issues/PRs of any repository in which the GitForWindowsHelper GitHub App is installed.
+
+**What does it do?** The app responds in a separate comment, saying "Hi @&lt;login&gt;" where `<login>` is your GitHub login name.
+
+### `/add release note <type> <message>`
```
Comment (rimrul) [inline README.md:22]: Do we also want to document the alternate spellings (`/add relnote`) and the invocation without `<type>`/`<message>`?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,56 @@
...
+
+**Where can it be called?** In `git-for-windows/git`'s [Pull Requests](https://github.com/git-for-windows/git/pulls)
+
+**What does it do?** This command starts [the `Git artifacts` Azure Pipeline](https://dev.azure.com/git-for-windows/git/_build?definitionId=34&_a=summary) that builds all of the artifacts of a full Git for Windows release: installer, Portable Git, MinGit, etc
+
+### `/release`
+
+**Where can it be called?** In `git-for-windows/git`'s [Pull Requests](https://github.com/git-for-windows/git/pulls)
+
+**What does it do?** Call this command after a successful `/git-artifacts` command successfully produced the artifacts _and_ after the installer artifact has been validated manually, using [the "pre-flight checklist"](https://github.com/git-for-windows/build-extra/blob/HEAD/installer/checklist.txt). This will start [the Azure Release Pipeline](https://dev.azure.com/git-for-windows/git/_release?_a=releases&view=mine&definitionId=1) to publish the artifacts in a new GitHub Release.
```
Comment (rimrul) [inline README.md:50]: The two `successful`s in close proximity read a little bumpy.

Comment (dscho) [reply README.md]: Fixed!

**Review by vdye — APPROVED** [review]

**Review by ldennington — APPROVED** [review]
Nice! 🚀

File: README.md [hunk 3]
```diff
@@ -7,6 +7,58 @@
...
+
+## Slash commands
+
+The GitForWindowsHelper GitHub App supports so-called "slash commands", i.e. commands that start with a forward slash and that are issued via comments in GitHub Issues or Pull Requests.
+
+### `/hi`
+
+**Where can it be called?** In issues/PRs of any repository in which the GitForWindowsHelper GitHub App is installed.
+
+**What does it do?** The app responds in a separate comment, saying "Hi @&lt;login&gt;" where `<login>` is your GitHub login name.
```
[the remark this answers is not in the document]
Comment (dscho) [reply README.md:20]: /hi

[the remark this answers is not in the document]
Comment (dscho) [reply README.md:20]: The idea of `/hi` is to be able to test that this works ;-)

**Pull request merged**